src/main/java/carracing/domain/entity/Winner.java

```diff
@@ -0,0 +1,33 @@
+package carracing.domain.entity;
+
+import java.util.Collections;
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+
+public class Winner {
+
+  private static final int FIRST_INDEX = 0;
+
+  private final List<Car> winnerList;
+
+  public Winner(List<Car> winnerList) {
+    this.winnerList = winnerList;
+  }
+
+  public List<Car> getWinnerList() {
+    return winnerList;
+  }
+
+  public static Winner pickWinner(List<Car> carList) {
+    Collections.sort(carList);
+    return new Winner(findWinner(carList));
+  }
+
+  private static List<Car> findWinner(List<Car> carList) {
+    Car firstCar = carList.get(FIRST_INDEX);
+    return carList.stream()
+                  .filter(car -> car.getNowStep() == firstCar.getNowStep())
```

Comment:
```suggestion
                  .filter(firstCar::equalStep)
```

이런식으로 step의 동등 비교에 대한 `메세지`를 자동차에게 전달하여 응답을 받으면 좀 더 객체지향적이지 않을까요?